### ISSUE TITLE
Ensure every image must have an alternative text for Accessibility

### ISSUE THREAD

**Issue by ipreencekmr** [expressjs/expressjs.com#1808]
Alt text plays a crucial role in accessibility by offering a textual description of images for individuals using screen readers. Each `<img>` element should include a meaningful alt attribute that accurately represents the image’s purpose or content. For decorative images, an empty **alt=""** should be used to ensure they are skipped by assistive technologies. Failing to provide appropriate alt text can hinder access for visually impaired users and lead to accessibility compliance issues.

**Severity:** High

**Reference:** https://dequeuniversity.com/rules/axe/4.10/image-alt?application=AxeChrome

<img width="1466" alt="Image" src="https://github.com/user-attachments/assets/b98e8f1a-64de-467f-91ea-6122215bd477" />

<img width="1449" alt="Image" src="https://github.com/user-attachments/assets/6dd340a8-c0f2-4813-a70b-8e786dbca08a" />

**Suggested Fix:**

Provide alt attribute to `<img>` tag with appropriate value.

**Help Required:**

- Translations needed for alt text

**Comment by ipreencekmr** [comment]
[PR](https://github.com/expressjs/expressjs.com/pull/1809) raised to fix

**Comment by bjohansebas** [comment]
Fixed by #1809
